runtime/debug/outer-port-attachment.js

```diff
@@ -20,16 +20,19 @@ export class OuterPortAttachment {
     this._particleRegistry = {};
   }
 
-  handlePecMessage(name, pecMsgBody, isReceiver) {
+  handlePecMessage(name, pecMsgBody, stackString) {
     // Skip speculative and pipes arcs for now.
     if (this._arcIdString.endsWith('-pipes') || this._speculative) return;
 
+    // Don't log the message used to set up stack trace passing from the context side.
+//    if (name === 'DevToolsConnected') return;
+
     const stack = [];
-    if (!isReceiver) {
+    if (stackString) {
       // The slice discards the first two stack frames corresponding to this
```

Comment:
Is discarding 2 top stack frames correct for both host and context?

Reply:
Yes, because it's the same basic call sequence through the API channel, whichever side of the fence you're on.